rpc/dist_ffi.go

```diff
@@ -0,0 +1,79 @@
+package rpc
+
+import (
+	"github.com/tchajed/marshal"
+	"io"
+	"net"
+)
+
+/// Sender
+type Sender struct {
+    conn net.Conn
+}
+
+func MakeSender(host string) *Sender {
+	// FIXME: cache "conn" in some global map to make connections live longer
```

Comment:
Benchmarking probably makes little sense until this is implemented -- but this goes beyond my Go skills.^^